reference/docs-conceptual/dev-cross-plat/create-help-using-platyps.md

```diff
@@ -0,0 +1,294 @@
+
+Once this step is complete, you will see `*-help.xml` and `about_*.help.txt` files in the target
+output folder.
+
+For more information, see [New-ExternalHelp][]
+
+### Test the compiled help files
+
+You can verify the content with the [Get-HelpPreview][] cmdlet:
+
+```powershell
+Get-HelpPreview -Path "<ModuleName>-Help.xml"
+```
+
+The cmdlet reads the compiled MAML file and outputs the content in the same format you would receive
+from `Get-Help`. This allows you to inspect the results to verify that the Markdown files compiled
+correctly and produce the desired results. If you find error, re-edit the Markdown files and
+recompile the MAML.
+
+Now you are ready to publish your help files.
+
+## Publishing your help
+
+Now that you have compiled the Markdown files into PowerShell help files, you are ready to make the
+files available to users. There are two options for providing help in the PowerShell console.
+
+- Package the help files with the module
+- Create an updateable help package that users install with the `Update-Help` cmdlet
+
+### Packaging help with the module
+
+The help files can be packaged with your module. See [Writing Help for Modules][] for details of the
+folder structure. You should include the list of Help files in the value of the **FileList** key in
+the module manifest.
+
+### Creating an updateable help package
+
+At a high level, the steps to create updateable help include:
+
+1. Find an internet site to host your help files
+1. Add a **HelpInfoURI** key to your module manifest
+1. Create a HelpInfo XML file
+1. Create CAB files
+1. Upload your files
+
+For more information, see [Supporting Updateable Help: Step-by-step][step-by-step].
+
+PlatyPS makes some of these steps easier.
+
+The **HelpInfoURI** is a URL that points to location where your help are hosted on the internet.
+This value is configured in the module manifest. `Update-Help` reads the module manifest to get this
+URL and download the updateable help content. This URL should only point to the folder location and
+not to individual files. `Update-Help` constructs the filenames to download based on other
+information from the module manifest and the HelpInfo XML file.
+
+> [!IMPORTANT]
+> The **HelpInfoURI** must end with a forward-slash character (`/`). Without that character,
+> `Update-Help` cannot construct the correct file paths to download the content. Also, most
+> HTTP-based files services are case-sensitive. It is important that the module metadata in the
+> HelpInfo XML file contains the proper letter case.
+
+The `New-ExternalHelp` cmdlet creates the HelpInfo XML file in the output folder. The HelpInfo XML
+file is built using YAML metadata contained in the module Markdown files (`ModuleName.md`).
+
+The `New-ExternalHelpCab` cmdlet creates ZIP and CAB files containing the MAML and
+`about_*.help.txt` files you compiled. CAB files are compatible with PowerShell 5.1 and older. Newer
+versions of PowerShell can use CAB or ZIP files.
```

Comment:
These last sentences read a bit funny. Can we just say CAB files are compatible with all versions of PowerShell and 6.0+ can use ZIP files or something along those lines?

Reply:
Fixed